src/main/java/com/google/sps/data/ComputeAPIClient.java

```diff
@@ -58,18 +58,17 @@
   private static final JsonFactory JSON_FACTORY = 
     JacksonFactory.getDefaultInstance();
   private static GoogleCredentials credential;
-  private  Compute compute;
+  private Compute compute;
+  private final String OPERATION_STATUS_DONE = "DONE";
 
-  /** 
-   * Creates an environment, calls each operation and tracks to see if they 
-   * are executed.
-   * @param {String} operation - the name of the method that should be called.
-   * @param {String} instanceName - the name of the instance being operated on.
-   * @param {Optional<String>} password - the password needed to access this 
-   *    instance. Not all operations requires a password.
+  /**
+   * Example of how to use computeManager
+   * ComputeAPIClientInterface example = new ComputeAPIClient();
+   * example.computeManager("createInstance", "vm1", Optional.of("12345"));
+   * example.computeManager("deleteInstance","testvm", null);
```

Comment:
Not great API design since anyone could put anything for the first argument. Restructure this code such that the usage is:
ComputeAPIClientInterface computeClient = new ComputeAPIClient();
computeClient.createInstance("vm1", blah);
computeClient.deleteInstance("testvm");